### PR TITLE
Add Croatian (hr) translations

### LINKED ISSUE #38734: Missing translations for Croatian (hr)
Hello,

There are some missing translation for Croatian (hr). These should be added in branch 4.4. 
See https://github.com/symfony/symfony/issues/38710 for details and [this page](https://symfony-translations.nyholm.tech/#pr) for an example.

These are the files that should be updated: 
- [src/Symfony/Component/Form/Resources/translations/validators.hr.xlf](https://github.com/symfony/symfony/blob/4.4/src/Symfony/Component/Form/Resources/translations/validators.hr.xlf)
- [src/Symfony/Component/Security/Core/Resources/translations/security.hr.xlf](https://github.com/symfony/symfony/blob/4.4/src/Symfony/Component/Security/Core/Resources/translations/security.hr.xlf)
- [src/Symfony/Component/Validator/Resources/translations/validators.hr.xlf](https://github.com/symfony/symfony/blob/4.4/src/Symfony/Component/Validator/Resources/translations/validators.hr.xlf)

NOTE: If you want to work on this issue, add a comment to assign it to yourself and let others know that this is already taken.

### PR DESCRIPTION
**Pull request by jseparovic1**
| Q             | A
| ------------- | ---
| Branch?       | 4.4
| Bug fix?      | no
| New feature?  |no 
| Deprecations? | no
| Tickets       | Fix #38734 
| License       | MIT

#SymfonyHackday

### CODE REVIEW

**Comment by Nyholm** [comment]
Awesome. 

Could I ask @HypeMC @MarioBlazek or @X-Coder264 to review this?

File: src/Symfony/Component/Form/Resources/translations/validators.hr.xlf [hunk 1]
```diff
@@ -14,6 +14,126 @@
...
+                <source>The values do not match.</source>
+                <target>Ove vrijednosti se ne poklapaju.</target>
+            </trans-unit>
+            <trans-unit id="119">
+                <source>Please enter a valid time.</source>
+                <target>Molim unesite ispravno vrijeme.</target>
+            </trans-unit>
+            <trans-unit id="120">
+                <source>Please select a valid timezone.</source>
+                <target>Molom odaberite ispravnu vremensku zonu.</target>
```
Comment (Nyholm) [inline src/Symfony/Component/Form/Resources/translations/validators.hr.xlf]: Could you double check this?

File: src/Symfony/Component/Validator/Resources/translations/validators.hr.xlf [hunk 2]
```diff
@@ -362,6 +362,30 @@
...
+                <source>This value is not a valid hostname.</source>
+                <target>Ova vrijednost ne nije ispravno ime poslužitelja (engl. hostname).</target>
+            </trans-unit>
+            <trans-unit id="96">
+                <source>The number of elements in this collection should be a multiple of {{ compared_value }}.</source>
+                <target>Broj elemenata u kolekciji treba biti djeljiv s {{ compared_value }}.</target>
+            </trans-unit>
+            <trans-unit id="97">
+                <source>This value should satisfy at least one of the following constraints:</source>
+                <target>Ova vrijednost mora zadovoljiti jedan od sljedećih ograničenja:</target>
```
Comment (Nyholm) [inline src/Symfony/Component/Validator/Resources/translations/validators.hr.xlf:379]: Does this say "limit" or "constraints"?

**Review by Nyholm — COMMENTED** [review]
Awesome. Thank you so much. 

I was using google translate to verify this. I might have been super wrong in my comments, but could you double check?

File: src/Symfony/Component/Form/Resources/translations/validators.hr.xlf (hunk 1 above)
Comment (jseparovic1) [reply src/Symfony/Component/Form/Resources/translations/validators.hr.xlf]: Good catch!

**Review by Nyholm — COMMENTED** [review]
Im happy with this PR. But I would be happy if a native speaker double checked.

File: src/Symfony/Component/Validator/Resources/translations/validators.hr.xlf (hunk 2 above)
Comment (jseparovic1) [reply src/Symfony/Component/Validator/Resources/translations/validators.hr.xlf:379]: It says constraints in my opinion.

File: src/Symfony/Component/Form/Resources/translations/validators.hr.xlf [hunk 3]
```diff
@@ -14,6 +14,126 @@
...
+                <source>Please enter a valid search term.</source>
+                <target>Molim unesite ispravan pojam za pretraživanje.</target>
+            </trans-unit>
+            <trans-unit id="123">
+                <source>Please provide a valid phone number.</source>
+                <target>Molim navedite ispravan telefonski broj.</target>
+            </trans-unit>
+            <trans-unit id="124">
+                <source>The checkbox has an invalid value.</source>
+                <target>Polje za potrvrdu sadrži neispravnu vrijednost.</target>
```
Comment (X-Coder264) [inline src/Symfony/Component/Form/Resources/translations/validators.hr.xlf]: @jseparovic1 "potvrdu"

File: src/Symfony/Component/Security/Core/Resources/translations/security.hr.xlf [hunk 4]
```diff
@@ -62,6 +62,14 @@
                 <source>Account is locked.</source>
                 <target>Račun je zaključan.</target>
             </trans-unit>
+            <trans-unit id="17">
+                <source>Too many failed login attempts, please try again later.</source>
+                <target>Previše neuspjelih pokušaja prijave, molim pokušajte ponovo kasnije.</target>
+            </trans-unit>
+            <trans-unit id="18">
+                <source>Invalid or expired login link.</source>
+                <target>Link za prijavu je istako ili je neispravan.</target>
```
Comment (X-Coder264) [inline src/Symfony/Component/Security/Core/Resources/translations/security.hr.xlf]: @jseparovic1 "istekao"

File: src/Symfony/Component/Validator/Resources/translations/validators.hr.xlf [hunk 5]
```diff
@@ -362,6 +362,30 @@
                 <source>This password has been leaked in a data breach, it must not be used. Please use another password.</source>
                 <target>Ova lozinka je procurila u nekom od sigurnosnih propusta, te je potrebno koristiti drugu lozinku.</target>
             </trans-unit>
+            <trans-unit id="94">
+                <source>This value should be between {{ min }} and {{ max }}.</source>
+                <target>Ova vrijednost treba biti između {{ min }} i {{ max }}.</target>
+            </trans-unit>
+            <trans-unit id="95">
+                <source>This value is not a valid hostname.</source>
+                <target>Ova vrijednost ne nije ispravno ime poslužitelja (engl. hostname).</target>
```
Comment (X-Coder264) [inline src/Symfony/Component/Validator/Resources/translations/validators.hr.xlf]: @jseparovic1 "ne" should be deleted from this sentence.

**Review by X-Coder264 — CHANGES_REQUESTED** [review]
@Nyholm Done. There's just some typos, the rest is fine.

**Comment by MarioBlazek** [comment]
Looks good. @jseparovic1 did a great job here. Address @X-Coder264 comments and that's it.

**Comment by Nyholm** [comment]
Awesome. Thank you. 

@jseparovic1 Just some more fixes and we are done here =)

**Comment by jseparovic1** [comment]
@MarioBlazek @X-Coder264 Thank you for review. 

@Nyholm Typos are fixed an PR is ready now 🚀

**Review by Nyholm — APPROVED** [review]
Thank you

**Comment by derrabus** [comment]
Thank you @jseparovic1.

**Pull request merged**